packages/@webex/plugin-cc/test/unit/spec/cc.ts

```diff
@@ -309,4 +316,84 @@ describe('webex.cc', () => {
       );
     });
   });
+
+  describe('getBuddyAgents', () => {
+    it('should return buddy agents response when successful', async () => {
+      const data: BuddyAgents = {state: 'Available', mediaType: 'telephony'};
+      webex.cc.agentConfig = {
+        agentId: 'agentId',
+        agentProfileId: 'test-agent-profile-id',
+      };
+
+      const buddyAgentsResponse: BuddyAgentsResponse = {
+        type: 'BuddyAgentsSuccess',
+        orgId: '',
+        trackingId: '1234',
+        data: {
+          eventType: 'BuddyAgents',
+          agentId: 'agentId',
+          trackingId: '1234',
+          orgId: '',
+          type: '',
+          agentSessionId: 'session123',
+          agentList: [
+            {
+              agentId: 'agentId',
+              state: 'Available',
+              teamId: 'teamId',
+              dn: '1234567890',
+              agentName: 'John',
+              siteId: 'siteId',
+            },
+          ],
+        },
+      };
+
+      const buddyAgentsMock = jest
```

Comment:
It's a spy not mock so please update the name accordingly, also please use toHaveBeenCalledOnceWith instead of toHaveBeenCalledWith in all the places

Reply:
Discussed 1:1